README.md

```diff
@@ -102,6 +102,10 @@ Following the aforementioned common instructions, install [Nativefier](https://g
 
 + The server will not send AJAX requests to One Codex's exposed URL endpoints if there is a proxy connection enabled in the browser. Since the desktop application is an [Electron](https://electronjs.org) wrapper over the web application, hence this limitation also holds for the desktop application as well.
 
+# Aim of the Project
+
++ The current paper reports that there are many more information that can be discovered from a microbiome sample. Microbiome Diversity Inspector can be used to highlight about the diversity of microbiome community in various materials found over different regions. The inclusion of other new algorithms in this platform can lead to interesting discoveries which may prove very beneficial to mankind
```

Comment:
@MAZHARMIK - Yes, it is better if we include the aim. Thanks for adding this.
Just a few suggestions - 
1) Add a period ('.') at the end, after the word - 'mankind' at the end of this section.
2) The phrase - 'current paper' seems irrelevant here, since the README is a documentation of the project, instead of a 'paper'. Let us replace the first line with something like - *'Presently, a lot of massive and groundbreaking initiatives are being undertaken to analyse microbiome data from diverse environments around the world in order to discover interesting secrets present in the world of microbiome.'* This is just a suggestion. Feel free to change the above line if you find some mistakes.

Reply:
Yes sure. Will do that soon.